src/components/pages/Dashboard/InstructorStats.js

```diff
@@ -1,70 +1,115 @@
+  StarOutlined,
 } from '@ant-design/icons';
-import { getStats } from '../../../redux/actions/instructorActions';
 import { useOktaAuth } from '@okta/okta-react';
-import { dummyData } from '../../../dummyData';
-
-const initialValues = [
-  {
-    icon: <UserOutlined style={{ fontSize: '40px', color: '#F79E1B' }} />,
-    title: 'My Students',
-    value: 0,
-  },
-  {
-    icon: <BarChartOutlined style={{ fontSize: '40px', color: '#F79E1B' }} />,
-    title: 'Active Course',
-    value: 0,
-  },
-  {
-    icon: <BarChartOutlined style={{ fontSize: '40px', color: '#F79E1B' }} />,
-    title: 'Completed Course',
-    value: 0,
-  },
-  {
-    icon: <DollarOutlined style={{ fontSize: '40px', color: '#F79E1B' }} />,
-    title: 'Total Earnings',
-    value: '$' + 0,
-  },
-];
+import { getCurrentUser } from '../../../redux/actions/userActions';
+import {
+  getInstructor,
+  getCourses,
+} from '../../../redux/actions/instructorActions';
 
-function InstructorStats() {
-  const { authState } = useOktaAuth();
+const initialValues = {
+  students: 0,
+  activeCourses: 0,
+  completedCourses: 0,
+};
+function InstructorStats(props) {
+  const { authState, authService } = useOktaAuth();
   const { idToken } = authState;
+  const dispatch = useDispatch();
   const [stats, setStats] = useState(initialValues);
 
   useEffect(() => {
-    // setStats(getStats()); // ENABLE WHEN READY TO CONNECT STATS TO STATE
-  }, []);
+    if (!props.user.name) {
+      dispatch(getCurrentUser(idToken, authState, authService));
+    }
+    // eslint-disable-next-line
+  }, [dispatch, idToken]);
+
+  useEffect(() => {
+    dispatch(getInstructor(idToken, props.user.profile_id));
+  }, [dispatch, idToken, props.user.profile_id]);
+
+  useEffect(() => {
+    dispatch(getCourses(idToken));
+  }, [dispatch, idToken]);
+
+  useEffect(() => {
+    let completed = 0;
+    let active = 0;
+    props.courses.forEach(course => {
+      if (course.active) {
+        active += 1;
+      } else {
+        completed += 1;
+      }
+      setStats({
+        ...stats,
+        completedCourses: completed,
+        activeCourses: active,
+      });
+    });
+    // eslint-disable-next-line
+  }, [props.courses]);
 
   const { Title } = Typography;
 
   return (
     <>
-      <Title className="instructor__name">
-        {dummyData.instructor_data.instructor_name}
-      </Title>
+      <Title className="instructor__name">{props.user.name}</Title>
       <div class="stats-wrapper">
-        {stats.map(stat => {
-          return (
-            <Card id="ant-card-stat">
-              <div class="stat-icon-wrapper">{stat.icon}</div>
-              <div class="stat-content">
-                <p>{stat.title}</p>
-                <h3>{stat.value}</h3>
-              </div>
-            </Card>
-          );
-        })}
+        <Card id="ant-card-stat">
+          <div class="stat-icon-wrapper">
+            <UserOutlined style={{ fontSize: '40px', color: '#F79E1B' }} />
```

Comment:
Is there a way to not have inline style? I know that Ant design has been giving us trouble, so no worries if not!